src/site/content/en/blog/content-visibility/index.md

```diff
@@ -113,6 +113,15 @@ As the element approaches the viewport, the browser no longer adds the `size`
 containment and starts painting and hit-testing the element's content. This
 enables the rendering work to be done just in time to be seen by the user.
 
+{% Aside 'caution' %}
+The browser is only able to skip rendering work if you are also careful not to call any DOM API that [forces some of rendering
+to occur](https://gist.github.com/paulirish/5d52fb081b3570c81e3a#file-what-forces-layout-md) on one of the skipped subtrees.
+If you're using `content-visibility` to improve performance, audit your code to make sure these APIs are not getting called.
+To help find them, Chromium will print console warnings if you call one of these APIs for a subtree of an
+element with `content-visibility:hidden`. To see the warnings, [turn on verbose logging](https://developer.chrome.com/docs/devtools/console/log/).
+{% endAside %}
+
+
```

Comment:
```suggestion
{% Aside 'caution' %}
The browser is only able to skip rendering work if you are also careful not to call any DOM API that [forces some of rendering
to occur](https://gist.github.com/paulirish/5d52fb081b3570c81e3a#file-what-forces-layout-md) on one of the skipped subtrees.
If you're using `content-visibility` to improve performance, audit your code to make sure these APIs are not getting called.
To help find them, Chromium will print console warnings if you call one of these APIs for a subtree of an
element with `content-visibility: hidden;`. To see the warnings, [turn on verbose logging](https://developer.chrome.com/docs/devtools/console/log/).
{% endAside %}

```

Reply:
I can't see what is different with this suggestion. Is anything different?